guessingGame.js

```diff
@@ -0,0 +1,23 @@
+function GuessingGame(answer) {
+  this.answer = answer;
+  this.recentGuess = null
```

Comment:
In JS, you can declare a variable without assigning a value if you don't have a value for it yet!